src/composables/accounts.ts

```diff
@@ -1,12 +1,23 @@
 import { computed } from '@vue/composition-api';
 import type { IAccount, IDefaultComposableOptions } from '../types';
+import { useMultisigAccounts } from './multisigAccounts';
 
 export function useAccounts({ store }: IDefaultComposableOptions) {
+  const {
+    isMultisigDashboard,
+    activeMultisigAccount,
+  } = useMultisigAccounts({ store });
   // TODO in th future the state of the accounts should be stored in this composable
   const accounts = computed((): IAccount[] => store.getters.accounts);
   const account = computed((): IAccount => store.getters.account);
   const accountsAddressList = computed(() => accounts.value.map((acc) => acc.address));
 
+  const activeAccountAddress = computed((): string => (
+    isMultisigDashboard.value && activeMultisigAccount.value
+      ? activeMultisigAccount.value.address
+      : account.value.address
+  ));
```

Comment:
I strongly suggest to not mix the regular accounts with multisig vaults. These are 2 separate features. Also there is no such a thing as `multisigAccount.address`.